task/buildah/0.1/buildah.yaml

```diff
@@ -281,6 +268,18 @@ spec:
       name: varlibcontainers
     workingDir: $(workspaces.source.path)
 
+  - name: upload-sbom
+    image: ghcr.io/sigstore/cosign/cosign:03468a1498876ccfb72e06f63fdb4f1c562dbca1
```

Comment:
Where does the image spec come from? I found this: https://github.com/sigstore/cosign#dockerfile

And they use this in an example: `gcr.io/projectsigstore/cosign:v1.13.0`

In any case, I know it's not commonly done, but I think it would be nice to have a comment to say where that image comes from.

Reply:
I took it from https://github.com/sigstore/cosign/pkgs/container/cosign%2Fcosign